src/pages/Hero/styles.ts

```diff
@@ -1,5 +1,31 @@
 import styled from 'styled-components'
+import { theme } from '../../styles'
 
 export const Content = styled.div`
+    
+`
 
+export const InfoCard = styled.div`
+    align-items: flex-start;
+    background-color: ${theme.background2};
+    border-radius: 4px;
+    display: flex;
+    flex-direction: column;
+    gap: 16px;
+    padding: 16px 16px 0 16px;
+    position: absolute;
+    width: 40%;
+`
+
+export const Text = styled.p`
+    color: ${theme.colors.black};
+    line-height: 2vh;
+
+    strong {
+        color: ${theme.colors.primaryRed};
```

Comment:
utilizar a prop do theme
```suggestion
        color: ${(props) => props.theme.colors.primaryRed};
```

Reply:
Done